js/readme.md

```diff
@@ -0,0 +1,22 @@
+names of the fields for the called by the id from the html are :

```

Comment:
Please make this documentation file not so ugly.

Reference this to polish it up
https://github.com/ricval/Documentacion/blob/master/Guias/GitHub/mastering-markdown.md